nexus/types/src/internal_api/params.rs

```diff
@@ -215,15 +217,13 @@ impl std::fmt::Debug for Certificate {
 
 #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
 pub struct RackInitializationRequest {
+    /// Services on the rack which have been created by RSS.
     pub services: Vec<ServicePutRequest>,
+    /// Datasets on the rack which have been provisioned by RSS.
     pub datasets: Vec<DatasetCreateRequest>,
-    // TODO(https://github.com/oxidecomputer/omicron/issues/1530):
-    // While it's true that Nexus will only run with a single address,
-    // we want to convey information about the available pool of addresses
-    // when handing off from RSS -> Nexus.
-
-    // TODO(https://github.com/oxidecomputer/omicron/issues/1528):
-    // Support passing x509 cert info.
+    /// IP pools which may be used for internal services, such as Nexus.
+    pub internal_services_ip_pool: Vec<IpRange>,
```

Comment:
This is a single pool, right? If so, the comment is a bit confusing. I might say, "IP ranges that form the pool used for internal services, such as Nexus", or something to that effect. Or might this need to be multiple pools?